tests/unit/type/set.tcl

```diff
@@ -177,23 +179,23 @@ foreach type {single multiple single_multiple} {
 
     test "Set encoding after DEBUG RELOAD" {
         r del myintset
-        r del myhashset
+        r del myhashtable
         r del mylargeintset
         r del mysmallset
         for {set i 0} {$i <  100} {incr i} { r sadd myintset $i }
         for {set i 0} {$i < 1280} {incr i} { r sadd mylargeintset $i }
         for {set i 0} {$i <   50} {incr i} { r sadd mysmallset [format "i%03d" $i] }
-        for {set i 0} {$i <  256} {incr i} { r sadd myhashset [format "i%03d" $i] }
+        for {set i 0} {$i <  256} {incr i} { r sadd myhashtable [format "i%03d" $i] }
         assert_encoding intset myintset
         assert_encoding hashtable mylargeintset
         assert_encoding listpack mysmallset
-        assert_encoding hashtable myhashset
+        assert_encoding hashtable myhashtable
```

Comment:
This test case used "myhashset" even with a dict? All sets here end in "set". I think it just means a set backed by a hash table, so it's not wrong.

Reply:
```suggestion
        assert_encoding hashtable myhashset
```